### ISSUE TITLE
Malidup missing alignments

### ISSUE THREAD

**Issue by mortonjt** [mortonjt/deep_blast_training#1]
Charlie noticed two types of alignment file issues.

1. occasionally there are two manual alignments in the malidup directory.  I'm not sure how to handle these.

2.  there is no entry for some malidup directories in your list of alignment.  Maybe these cull for some reason like they were too short or they were accidentally in your training set or test set.

here's a partial list of some f each of these:

8       d1ahua1
16      d1anv_2
56      d1d5ya1
59      d1df9c_
72      d1efud4
84    d1fga___2

EDIT: the updated list is below
` [['d1anv_2', '1anv'], ['d1d5ya1', '1d5y'], ['d1df9c_', '1df9'], ['d1fga___2', '1fga'], ['d1hg7a_', '1hg7'], ['d1kg1a_', '1kg1'], ['d1lpba1', '1lpb']]`

EDIT2 : below is another edge case
```
/home/cems/Documents/JamieMorton/mammothalgn/malidup/d1ahua1/1ahu_2.manual2.ali
/home/cems/Documents/JamieMorton/mammothalgn/malidup/d1efud4/1efu_2.manual2.ali
/home/cems/Documents/JamieMorton/mammothalgn/malidup/d1i31a_/1i31_2.manual2.ali

those directories all have a two variants of the manual alignment.  Your data base seems to use the secondary one when there are two not the primary. 
most directories have just a single manual alignment names XXXXX.manual.ali  instead of manual2.ali
```

**Comment by mortonjt** [comment]
Ok, I've investigated into this a bit.  Entries like ` d1ahua1` appear to have 2 manual alignments.  For these scenarios, we are randomly picking one of the manual alignments as ground truth (its arbitrary at the moment).

Entries such as `d1anv_2`, don't have a mammoth alignment, so we are ignoring these at the moment.

**Comment by mortonjt** [comment]
Ok, it turns out that I was goofing up the parsers.  This is now being fixed in https://github.com/flatironinstitute/deepblast/pull/79
